### ISSUE TITLE
Search configuration

### ISSUE THREAD

**Issue by matlae-ugly** [scandipwa/base-theme#1231]
**Description**:
Magento is pretty vague in their docs about Elasticsearch... I understnad we use it... they also tell we can use it for EAV indexing... they tell we can specify some parameters like minimum and max query length... 

I think we should just describe what we got.. how do we set it up.. container? What version do we use? What paameters we do respect from magento BE? Will we show search queries history? 

Basically, this ticket requires more like description of our solution + you ofc can inish it, if you see that some important things, but not taking ages can be done. 

Once described - showcase!

**Comment by matlae-ugly** [comment]
The issue was resolved in another project management software. Transferred to GitHub for full transparency.
